config/locales/en.yml

```diff
@@ -38,8 +38,10 @@ en:
   components:
     search_bar:
       placeholders:
-        users: 'Search for users'
-        projects: 'Search for projects'
+        users: "Search for users"
```

Comment:
can you add translation in hindi and other languages too?
Arabic is not the only we want!

Reply:
will be handled by crowdin, as @tachyons suggested